jfrog_platform_self_hosted/assets/dashboards/jfrog_artifactory_logs_(self-hosted).json

```diff
@@ -7,13 +7,13 @@
             "available_values": [],
             "default": "*",
             "name": "ip_address",
-            "prefix": "@ip_address"
+            "prefix": "@network.client.ip"
         },
         {
             "available_values": [],
             "default": "*",
             "name": "username",
-            "prefix": "@user_name"
+            "prefix": "@usr.id"
```

Comment:
This will update the dashboard for all users. To confirm, you would like to continue with this change?